package.json

```diff
@@ -42,7 +43,10 @@
 	},
 	"dependencies": {
 		"@floating-ui/dom": "^1.5.3",
+		"@protobuf-ts/grpcweb-transport": "^2.9.1",
+		"@protobuf-ts/runtime": "^2.9.1",
 		"@tauri-apps/api": "^1.5.1",
+		"chalk": "^5.3.0",
```

Comment:
This is not a runtime dependency. Please also check that the other protobuf dependencies are actually runtime dependencies. ( I dont know )

Reply:
The other dependencies are for the generated code